internal/providers/terraform/ibm/testdata/is_share_test/is_share_test.usage.yml

```diff
@@ -0,0 +1,8 @@
+version: 0.1
+resource_usage:
+  ibm_is_share.testNFS:
+    is-share_monthly_instance_hours: 730 # Monthly number of instance hours
+
+  ibm_is_share.testNFS2withInlineReplica:
+    is-share_monthly_instance_hours: 730 # Monthly number of instance hours
+    is-share_monthly_transmitted_gb: 1   # Data transmitted to the instance
```

Comment:
Did we still wanna follow the naming convention `<SERVICE_PROGRAMMATIC_NAME>_<CHARGE_UNIT_NAME>` for usage variables, for compatibility with Demis' work?